examples/usecases/transformers-next-item-prediction.ipynb

```diff
@@ -87,7 +87,7 @@
      "name": "stderr",
```

Comment:
<p>Thanks. I think since you are already filtering them in the NVT pipeline, using "features &gt;&gt; ops.Filter(f=lambda df: df["city_id_count"] &gt;= MINIMUM_SESSION_LENGTH)"  not sure we need to filter them here as well.</p>